### DISCUSSION TITLE
[Feature request] Add support for Kotlin/JS NPM package manager

### DISCUSSION THREAD

**Discussion by krzema12** [renovatebot/renovate#23309] [Suggest an Idea]
### Tell us more.

See https://kotlinlang.org/docs/using-packages-from-npm.html - in Kotlin/JS, it's possible to add a dependency on an NPM package within the Gradle script, and Gradle will fetch it:

```kotlin
dependencies {
    // ...
    implementation(npm("is-sorted", "1.0.5"))
}
```

Would you be open to having support for it implemented in Renovate? If yes, I'll fill in the [questionnaire](https://github.com/renovatebot/renovate/blob/main/docs/development/new-package-manager-template.md).

**Comment by viceice** [comment]
yes

**Reply by viceice** [reply]
I think this should be live inside Gradle manager.

**Comment by Churro** [comment]
Duplicate of https://github.com/renovatebot/renovate/issues/19920
